include/deal.II/distributed/grid_tools.h

```diff
@@ -79,6 +79,39 @@ namespace parallel
      *   on the sending side returned a non-empty boost::optional object.
      *   The @p unpack function is then called with the data sent by the
      *   processor that owns that cell.
+     *
+     *
+     * <h4> An example </h4>
+     *
+     * Here is an example that shows how this function is to be used
+     * in a concrete context. It is taken from the code that makes
+     * sure that the @p active_fe_index (a single unsigned integer) is
+     * transported from locally owned cells where one can set it in
+     * hp::DoFHandler objects, to the corresponding ghost cells on
+     * other processors to ensure that one can query the right value
+     * also on those processors:
+     * @code
+     *    auto pack
+     *    = [] (const typename dealii::hp::DoFHandler<dim,spacedim>::active_cell_iterator &cell) -> unsigned int
+     *    {
+     *      return cell->active_fe_index();
+     *    };
+     *
+     *    auto unpack
+     *      = [] (const typename dealii::hp::DoFHandler<dim,spacedim>::active_cell_iterator &cell,
+     *            const unsigned int                                                        &active_fe_index) -> void
+     *    {
+     *      cell->set_active_fe_index(active_fe_index);
```

Comment:
No longer, see #4817. The code is actually copied verbatim from `source/hp/dof_handler.cc`